### PR TITLE
book uploads: support pdf chapters 

### PR DESCRIPTION
**Pull request by umutacar**
* Allows pdf chapters
* Allows specifying labels, video url, title, etc
Example json
```
{
  "book" : "SLIDES",
  "chapters": [
    {
       "number": 1,
       "title": "Introduction",
       "label": "ch:introduction",
       "pdf": "slides-introduction.pdf"
    },
    {
       "number": 2,
       "title": "Recurrences",
       "label": "ch:recurrences",
       "pdf": "slides-recurrences.pdf"
    }   
  ]
}
```

### CODE REVIEW

File: standalone.py [hunk 1]
```diff
@@ -431,28 +431,35 @@ def adjust_search_path(path):
         if chapters is None:
             exit_with_error("invalid JSON: could not find field 'chapters'")
         for chapter in chapters:
+            # Extract data
```
Comment (rohany) [inline standalone.py:434]: [nit]: Replace with `Extract data from the chapter JSON.`

Comment (umutacar) [reply standalone.py]: done

**Comment by rohany** [comment]
Sorry, but please include the period at the end! Feel free to merge it after adding.

**Comment by umutacar** [comment]
ohh that is a little too much for me....

**Pull request merged**